City-of-Helsinki/drupal-hdbt
# UHF-5304: Added a template for video that can be reused in paragraph …

…or some other node that has a remote video field

# [UHF-5304](https://helsinkisolutionoffice.atlassian.net/browse/UHF-5304)

Video paragraph template cannot be reused for remote video fields

## What was done

Added a template for video that can be reused in the video paragraph or in some node that has remote video fields

## How to install

* Make sure your instance is up and running on latest dev branch.
  * `git pull origin dev`
  * `make fresh`
* Update the HDBT theme
  * `composer require drupal/hdbt:dev-UHF-5304-video-for-job-listing`
* Run `make drush-cr`

## How to test

Check https://github.com/City-of-Helsinki/drupal-helfi-rekry/pull/25 for instructions


## Designers review



## Other PRs


* https://github.com/City-of-Helsinki/drupal-helfi-rekry/pull/25
